index.html

```diff
@@ -0,0 +1,1752 @@
+                        alt=""
+                      />
+                    </div>
+                    <h4 class="h5">
+                      <a href="#"
+                        >'Friday Night SmackDown' Results (5/29): IC Tournament
+                        Continues</a
+                      >
+                    </h4>
+                  </div>
+                  <div class="col-lg-4 col-sm-12 col-md-4">
+                    <article class="d-flex pb-2 border-bottom">
+                      <div class="photo-1">
+                        <img src="resources/boston-marathon.jpg" alt="" />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >Boston Marathon Is Going Virtual, So What Does It
+                          Mean for Other Marathons?</a
+                        >
+                      </h4>
+                    </article>
+
+                    <article class="d-flex pt-2 pb-2 border-bottom">
+                      <div class="pic">
+                        <img
+                          class="pr-3"
+                          src="resources/kansas-city-chiefs-fans.jpg"
+                          alt=""
+                        />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >NFL Home-field Advantage Could Be Wiped Out By
+                          Playing Behind Closed Doors</a
+                        >
+                      </h4>
+                    </article>
+
+                    <article class="d-flex pt-2">
+                      <div class="pic">
+                        <img
+                          class="pr-3"
+                          src="resources/adidas-face-mask-press-image.jpg"
+                          alt=""
+                        />
+                      </div>
+                      <h4 class="h3">
+                        <a href="#"
+                          >Where to Buy Running Face Masks From Adidas, Nike,
+                          the NBA and More</a
+                        >
+                      </h4>
+                    </article>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="subscribe-book mt-3 mb-4 container">
+              <div class="row">
+                <div class="col-lg-7 col-sm-12 col-xl-7">
+                  <div class="block-title1 red">Subscribe</div>
+                  <div
+                    class="gray p-5 row justify-content-around align-items-center"
+                  >
+                    <img
+                      class="col-lg-4 col-md-4 col-sm-12"
+                      src="resources/may-29-june-06-2020-cover.jpg"
+                      alt=""
+                    />
+                    <div class="info col-lg-8 col-sm-12 col-md-8">
+                      <h3 class="subscribe-block">
```

Comment:
- [ ] Kindly add the propper layout for these images like on the source page. Dealing with absolute and relative positioning is a great tool that will help you in the future. You can also check the source page to see how they achieve it.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83825804-01005a80-a6a0-11ea-86f9-61e1a3a58983.png) | ![image](https://user-images.githubusercontent.com/57414789/83825781-f0e87b00-a69f-11ea-997a-e670f6960b30.png)